core/src/main/java/contacts/core/util/DefaultContactData.kt

```diff
@@ -12,12 +12,12 @@ import contacts.core.entities.table.Table
 /**
  * Returns the default data entity in the collection or null if not found.
  */
-fun Collection<CommonDataEntity>.default(): CommonDataEntity? = firstOrNull { it.isDefault }
+fun <T : DataEntity> Collection<T>.default(): T? = firstOrNull { it.isDefault }
```

Comment:
Just a random improvement. This extension now returns the actual receiver type instead of being potentially upcasted to a `DataEntity`.